semcore/feedback-form/src/component/slider-rating/SliderRating.tsx

```diff
@@ -96,21 +115,37 @@ class SliderRatingRoot extends Component<SliderRatingProps, State> {
     }
   };
 
+  getLabelText() {
+    const { hoveredIndex } = this.state;
+    const { readonly, value } = this.asProps;
+
+    if (readonly) {
+      return `Your rating: ${value} out of 5`;
+    }
+
+    return value ? value : hoveredIndex === -1 ? 'Not set' : `${hoveredIndex + 1} out of ${MAX}`;
+  }
+
   render() {
-    const { styles, value } = this.asProps;
+    const { styles, readonly, getI18nText, uid, noticeTextId } = this.asProps;
     const { hoveredIndex } = this.state;
+
     const SSliderRating = Root;
+    const sliderDescriberId = `${uid}-slider-describer`;
+    const role = readonly ? 'image' : 'slider';
 
-    const label = value ? value : hoveredIndex === -1 ? 'Not set' : hoveredIndex + 1;
+    const label = this.getLabelText();
 
     return sstyled(styles)(
       <SSliderRating
         render={Flex}
         gap={1}
         onMouseLeave={this.handleMouseLeave}
         onKeyDown={this.handleKeyDown}
-        role='slider'
+        role={role}
         aria-orientation='horizontal'
+        aria-describedby={sliderDescriberId}
+        aria-labelledby={noticeTextId}
```

Comment:
you don't need to add `noticeTextId` prop, just pass `aria-labelledby` prop to the `SliderRating` from FeedbackRating